misc/stackblitz-gen.js

```diff
@@ -110,10 +112,10 @@ import { AppComponent } from './app.component';
 import { ${demoImports} } from '${demoImport}';
 
 @NgModule({
-  imports: [BrowserModule, FormsModule, ReactiveFormsModule, HttpClientModule, NgbModule.forRoot()], 
+  imports: [BrowserModule, FormsModule, ReactiveFormsModule, HttpClientModule, NgbModule.forRoot()],
```

Comment:
There are ws changes in this file and others - why?

BTW, I know it is not related to your change, but why do we have `NgbModule.forRoot()` here?

Reply:
We never updated it i think... I will remove them